azure/deploy-pipeline.yml

```diff
@@ -10,12 +10,11 @@ jobs:
         inputs:
           workingDirectory: ''
           gradleWrapperFile: 'gradlew'
+          options: '-Dorg.gradle.internal.http.socketTimeout=120000 -Dorg.gradle.internal.http.connectionTimeout=120000'
           gradleOptions: '-Xmx3072m'
           javaHomeOption: 'JDKVersion'
           jdkVersionOption: '1.8'
           jdkArchitectureOption: 'x64'
-          publishJUnitResults: true
-          testResultsFiles: '**/TEST-*.xml'
```

Comment:
I missed it earlier but I think removing these breaks the test output in Azure

Reply:
The publish tasks don't run tests, right?

Reply:
oh right nevermind